README.md

```diff
@@ -19,3 +21,41 @@ composer require pantheon-systems/edge-integrations-wordpress-sdk
 ```
 
 That command will add this repository to your `/vendor` directory, as well as all of the dependencies, which include a [global, CMS-agnostic PHP library](https://github.com/pantheon-systems/pantheon-edge-integrations) and a [WordPress plugin](https://github.com/pantheon-systems/pantheon-wordpress-edge-integrations) as well as all of the documentation for the SDK.
+
+#### Step 2: Require the Composer autoloader
+
+You'll need to make sure your WordPress project is loading the Composer autoloader (`vendor/composer/autoload.php`). One way to do this is in a `mu-plugins` loader file like [this one](https://github.com/pantheon-systems/wordpress-bedrock-recommended/blob/master/packages/pantheon-wp-loader/loader.php) <!-- TODO: Update this link when this is broken out into a new repository -->, but other methods exist, such as adding an `include_once __DIR__ . '/vendor/composer/autoload.php` line in your `wp-config.php` or other file that runs before WordPress is executed.
+
+### Installing manually
+
+If you aren't running a project with Integrated Composer but you _do_ use Composer otherwise, you can still get started with the Edge Integrations WordPress SDK without too much trouble. This still assumes you have Composer installed on at least one machine.
+
+#### Step 1: Clone the WordPress Edge Integrations plugin
+
+First, you'll need to get a copy of the [WordPress Edge Integrations plugin](https://github.com/pantheon-systems/pantheon-wordpress-edge-integrations). Clone or download the plugin into your `wp-content/plugins` directory.
+
+```bash
+git clone git@github.com:pantheon-systems/pantheon-wordpress-edge-integrations.git
+```
+
```

Comment:
Not sure if it's worth including but the plugin can also be pulled from packagist using
`composer require pantheon-systems/pantheon-wordpress-edge-integrations`

Reply:
I suppose I could add it as a note since this still assumes that Composer is installed on the client machine.